cpp/CMakeLists.txt

```diff
@@ -461,30 +217,32 @@ target_compile_definitions(cugraph PUBLIC CUDA_API_PER_THREAD_DEFAULT_STREAM)
 # - include paths ---------------------------------------------------------------------------------
 target_include_directories(cugraph
     PRIVATE
-    "${THRUST_INCLUDE_DIR}"
-    "${CUCO_INCLUDE_DIR}"
-    "${LIBCUDACXX_INCLUDE_DIR}"
-    "${CMAKE_CUDA_TOOLKIT_INCLUDE_DIRECTORIES}"
-    "${Boost_INCLUDE_DIRS}"
-    "${RMM_INCLUDE}"
-    "${CMAKE_CURRENT_SOURCE_DIR}/../thirdparty"
-    "${CUHORNET_INCLUDE_DIR}/hornet/include"
-    "${CUHORNET_INCLUDE_DIR}/hornetsnest/include"
-    "${CUHORNET_INCLUDE_DIR}/xlib/include"
-    "${CUHORNET_INCLUDE_DIR}/primitives"
-    "${CMAKE_CURRENT_SOURCE_DIR}/src"
-    "${GUNROCK_INCLUDE_DIR}"
-    "${NCCL_INCLUDE_DIRS}"
-    "${RAFT_DIR}/cpp/include"
+        "${CMAKE_CURRENT_SOURCE_DIR}/../thirdparty"
+        "${CMAKE_CURRENT_SOURCE_DIR}/src"
+        "${NCCL_INCLUDE_DIRS}"
     PUBLIC
-    "${CMAKE_CURRENT_SOURCE_DIR}/include"
+        "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>"
+        "$<INSTALL_INTERFACE:include/cugraph>"
```

Comment:
@trxcllnt With your reworking of the cugraph includes should this now be just `include/`?

Reply:
Yes, I think so